src/main/kotlin/blackjack/model/FinalResult.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.model
+
+import blackjack.controller.Controller.Companion.BLACKJACK_SCORE
+
+class FinalResult(val dealer: Dealer, players: List<Player>) {
```

Comment:
```suggestion
class FinalResult(dealer: Dealer, players: List<Player>) {
```

I think we can remove the `val` here :)